python/create_account_with_captcha.py

```diff
@@ -30,36 +33,48 @@
 
 @APP.route("/", methods=['GET', 'POST'])
 def show_form():
-    """ Render form template and handle form submission request
-    """
+    """ Render form template and handle form submission request """
 
     captcha_fields = get_captcha_fields()
-    captcha_url = WIKI_URL + captcha_fields['captchaInfo']['value']
+    captcha_url = TEST_WIKI_URL + captcha_fields['captchaInfo']['value']
 
     if request.method == 'POST':
         details = {
             'name': request.form['username'],
             'password': request.form['password'],
-            'confirm_password': request.form['confirm-password'],
+            'confirm_password': request.form['retype'],
             'email': request.form['email'],
             'captcha_word': request.form['captcha-word'],
-            'captcha_id': captcha_fields['captchaId']['value']
-        }
+            'captcha_id': captcha_fields['captchaId']['value']}
 
         create_account(details)
 
+    register_fields = list()
+    registration_fields = get_registration_fields()
+    for x in registration_fields:
```

Comment:
Variable names in this for loop could be slightly improved and made more self-explanatory. We don't want them to be lengthy either..